warmup.py

```diff
@@ -240,17 +254,36 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        points = np.atleast_2d(points)
+        for i in range(len(points[:-1])):
+            for j in range(len(points[i+1:])):
```

Comment:
To simplify this, you can use range with `start=i+1` and `stop=points.shape[0]`. Then, you can just use i and j directly below.

https://www.geeksforgeeks.org/python-range-function/